Directory.Build.targets

```diff
@@ -121,26 +136,21 @@ or
     <ItemGroup>
       <CocoaDeviceBuildPath Include="$(SentryCocoaRoot)Carthage/Build/Sentry.xcframework/ios-arm64_armv7/Sentry.framework/**/*" />
     </ItemGroup>
-    <Copy SourceFiles="@(CocoaDeviceBuildPath)"
-          DestinationFiles="@(CocoaDeviceBuildPath->'$(SentryCocoaDeviceArtifactsDestination)%(RecursiveDir)%(Filename)%(Extension)')">
+    <Copy SourceFiles="@(CocoaDeviceBuildPath)" DestinationFiles="@(CocoaDeviceBuildPath->'$(SentryCocoaDeviceArtifactsDestination)%(RecursiveDir)%(Filename)%(Extension)')">
     </Copy>
     <ItemGroup>
       <CocoaSimulatorBuildPath Include="$(SentryCocoaRoot)Carthage/Build/Sentry.xcframework/ios-arm64_i386_x86_64-simulator/Sentry.framework/**/*" />
     </ItemGroup>
-    <Copy SourceFiles="@(CocoaSimulatorBuildPath)"
-          DestinationFiles="@(CocoaSimulatorBuildPath->'$(SentryCocoaSimulatorArtifactsDestination)%(RecursiveDir)%(Filename)%(Extension)')">
+    <Copy SourceFiles="@(CocoaSimulatorBuildPath)" DestinationFiles="@(CocoaSimulatorBuildPath->'$(SentryCocoaSimulatorArtifactsDestination)%(RecursiveDir)%(Filename)%(Extension)')">
     </Copy>
 
-    <Error Condition="(!Exists('$(SentryCocoaDeviceArtifactsDestination)') OR !Exists('$(SentryCocoaSimulatorArtifactsDestination)'))"
-           Text="Failed to build the Cocoa SDK.">
+    <Error Condition="(!Exists('$(SentryCocoaDeviceArtifactsDestination)') OR !Exists('$(SentryCocoaSimulatorArtifactsDestination)'))" Text="Failed to build the Cocoa SDK.">
     </Error>
   </Target>
 
   <!-- Build the Android SDK: dotnet msbuild /t:BuildAndroidSDK src/Sentry.Unity -->
-  <Target Name="BuildAndroidSDK"
-          Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'
-          And !Exists('$(SentryAndroidArtifactsDestination)')"
-          BeforeTargets="BeforeBuild">
+  <Target Name="BuildAndroidSDK" Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'
```

Comment:
This is just a style change? Arguably harder to read. Or at least harder to review the changes as not easy to tell changes in style vs behavior